### PR TITLE
Sphinx doc chapter 1

### PR DESCRIPTION
**Pull request by maximedenes**

### CODE REVIEW

File: doc/sphinx/language/gallina-specification-language.rst [hunk 1]
```diff
@@ -1,1736 +1,1323 @@
...
+as theorems) defined within the section and depending on :token:`ident` are
+prefixed by the let-in definition ``let`` :token:`ident` ``:=`` :token:`term`
+``in``. Using the ``Let`` command out of any section is equivalent to using
+``Local Definition``.
+
+.. exn:: @ident already exists
+
+.. cmdv:: Let @ident : @term := @term.
+
+.. cm
```
Comment (Zimmi48) [inline doc/sphinx/language/gallina-specification-language.rst]: Ah ah :laughing:

Comment (Zimmi48) [inline doc/sphinx/language/gallina-specification-language.rst]: Is this a choice not to keep `.. example::`?

Comment (Zimmi48) [inline doc/sphinx/language/gallina-specification-language.rst]: It seems to me that this should not start with a capital letter, or be reformulated to be moved above.

**Review by Zimmi48 — COMMENTED** [review]
Looks very good. I only have minor remarks.

**Comment by Zimmi48** [comment]
The last few remarks only show on the correct lines if you look to them in the context of commit 4fea4af460939b41b6b1ea9624db2de4291684aa

**Comment by Zimmi48** [comment]
@maximedenes Can you look at my comments on 4fea4af and decide whether you want to address them or not? Could you also (whatever you decide) rebase the current PR on top of #7117?

**Comment by Zimmi48** [comment]
@maximedenes ping

**Comment by maximedenes** [comment]
I addressed all comments.

**Pull request merged**